templates/registration/register.html.twig

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Register{% endblock %}
+
+{% block stylesheets %}
+    {{ encore_entry_link_tags('login_style') }}
+{% endblock %}
+
+{% block body %}
+    <div class="container">
+        <div class="row justify-content-center sign-container">
+            <div class="col-xs-12 col-sm-6 col-md-4">
+                <h1 class="text-center login-title">Registration</h1>
+                <div class="account-wall">
+                    <img class="profile-img" src="{{ asset('build/images/login/add-user-icon-256.png') }}"
+                         alt="">
+                    {% for flashError in app.flashes('verify_email_error') %}
+                        <div class="alert alert-danger" role="alert">{{ flashError }}</div>
+                    {% endfor %}
+
+                    {{ form_start(registrationForm, {'attr': {'class': 'form-signin'}}) }}
+                    {{ form_widget(registrationForm.firstName, {'attr': {'class': 'form-control',
+                        'placeholder': 'Firstname'}}) }}
+                    {{ form_errors(registrationForm.firstName) }}
+                    {{ form_widget(registrationForm.lastName, {'attr': {'class': 'form-control',
+                        'placeholder': 'Lastname'}}) }}
+                    {{ form_errors(registrationForm.lastName) }}
+                    {{ form_widget(registrationForm.email, {'attr': {'class': 'form-control',
+                        'placeholder': 'Email'}}) }}
+                    {{ form_errors(registrationForm.email) }}
+                    {{ form_widget(registrationForm.password.first, {'attr': {'class': 'form-control',
+                        'placeholder': 'Password'}}) }}
+                    {{ form_widget(registrationForm.password.second, {'attr': {'class': 'form-control',
+                        'placeholder': 'Repeat password'}}) }}
+
+                    <button type="submit" class="btn btn-lg btn-dark btn-block">Register</button>
```

Comment:
Add `SubmitType` to form just like another field.

Reply:
Resolve